lib/exams/matematika-5-2023-1.ts

```diff
@@ -1,6 +1,58 @@
-import { group, number } from "../utils/quiz-builder";
+
+
+import { group, number, option, optionBool, selfEvaluateImage, task3Max4Points, task3Max5Points, twoPoints } from "../utils/quiz-builder";
 
 const form = group({
-  1: number(20, { suffix: 'minut' }),
+  1: group({
+    1.1: number(710, {}, twoPoints),
+    1.2: number(126, {}, twoPoints),
+  }),
+  2: group({
+    2.1: number(2, { prefix: 'o', suffix: 'litry' }, twoPoints),
+    2.2: number(60, {}, twoPoints)
+
+  }),
+  3: group({
+    3.1: number(120, { suffix: 'vojínů' }),
+    3.2: number(17, { suffix: 'osob' }),
+    3.3: number(136, { suffix: 'osob' }),
+  }),
+  4: group({
+    4.1: number(27, { suffix: 'linkovaných sešitů ' }),
+    4.2: number(54, { suffix: 'korun' }, twoPoints),
+    4.3: number(520, { suffix: 'korun' }, twoPoints),
+  }),
+  5: group({
+    5.1: number(7, { suffix: 'dětí' }, twoPoints),
+    5.2: number(18, { suffix: 'dětí' }, twoPoints),
+  }),
+  6: group({
+    6.1: number(2, { suffix: 'krát více' }),
+    6.2: number(900, { suffix: 'korun' }, twoPoints),
+  }),
+  7: group({
+    7.1: selfEvaluateImage(3, "primky ab v rrovine vysledrk"),
+    7.2: selfEvaluateImage(3, "V rovině leží body K, S a přímka p procházející bodem S."),
```

Comment:
image name - do not use spaces and diacriticts

Reply:
the number 3 must be there?